2.8.0 throws "Rest parameter 'args' implicitly has an 'any[]'"

<!-- Please replace all placeholders such as this below -->

**What happened?**

When upgrading from `2.7.1` to `2.8.0` starts throwing the following compile error:

```bash
> gcf-utils@2.0.0 compile /Users/bencoe/google/repo-automation-bots/packages/gcf-utils
> tsc -p .

node_modules/@octokit/request/node_modules/@octokit/types/src/GetResponseTypeFromEndpointMethod.ts:2:21 - error TS7019: Rest parameter 'args' implicitly has an 'any[]' type.

2 type AnyFunction = (...args) => any;
                      ~~~~~~~


Found 1 error.
```

**What did you expect to happen?**

`tsc` to compile the code 😆 

<!-- Describe what you expected to happen instead -->

**What the problem might be**

Computers 💻 

<!-- If you have an idea where the bug might lie, please share here. Otherwise remove the whole section -->

--- a/src/GetResponseTypeFromEndpointMethod.ts
+++ b/src/GetResponseTypeFromEndpointMethod.ts
@@ -1,5 +1,5 @@
 type Unwrap<T> = T extends Promise<infer U> ? U : T;
-type AnyFunction = (...args) => any;
+type AnyFunction = (...args: any[]) => any;
 
 export type GetResponseTypeFromEndpointMethod<T extends AnyFunction> = Unwrap<
   ReturnType<T>

